src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -4557,22 +4618,50 @@ namespace AirflowNetworkBalanceManager {
 			if ( AirflowNetworkNodeData( n ).NodeTypeNum == 0 ) {
 				AirflowNetworkNodeSimu( n ).PZ = 0.0;
 			} else {
+				// NodeTypeNum = 1, external nodes
 				// Assigning ambient conditions to external nodes
+
+				// X LUO Added Aug 2017
+				// Use local node
+
 				i = AirflowNetworkNodeData( n ).ExtNodeNum;
 				if ( i > 0 ) {
-					if ( i <= AirflowNetworkNumOfExtNode ) {
-						Vref = WindSpeedAt( MultizoneExternalNodeData( i ).height );
-						AirflowNetworkNodeSimu( n ).PZ = CalcWindPressure( MultizoneExternalNodeData( i ).curve,
-							Vref, AirflowNetworkNodeData( n ).NodeHeight, MultizoneExternalNodeData( i ).azimuth,
-							MultizoneExternalNodeData( i ).symmetricCurve, MultizoneExternalNodeData( i ).useRelativeAngle);
-						//AirflowNetworkNodeSimu( n ).PZ = CalcWindPressure( MultizoneExternalNodeData( i ).CPVNum, Vref, AirflowNetworkNodeData( n ).NodeHeight );
-					}
-					AirflowNetworkNodeSimu( n ).TZ = OutDryBulbTempAt( AirflowNetworkNodeData( n ).NodeHeight );
-					AirflowNetworkNodeSimu( n ).WZ = OutHumRat;
-				} else {
+					if ( AnyLocalEnvironmentsInModel && i <= AirflowNetworkNumOfExtNode && MultizoneExternalNodeData( i ).OutAirNodeNum != 0 ) {
```

Comment:
Thanks for your comments. Due to the complexity of the way Airflow Network uses the local air conditions, we decided to defer the implementation of the new local air node feature for airflow net work to a future release. We had discussed with Gu three times but need further discussions on an efficient implementation. Your inputs would be critical as well.